### PR TITLE
rec: prep 5.0.0 rc1

### PR DESCRIPTION
**Pull request by omoerbeek**
### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

File: pdns/recursordist/docs/appendices/EOL.rst [hunk 1]
```diff
@@ -3,29 +3,29 @@
 End of life statements
 ======================
 
-We aim to have a release every six months.
-The latest release receives correctness, stability and security updates.
-The two releases before that get critical updates only.
+We aim to have a major release every six months.
+The latest release receives correctness, stability and security updates by releasing minor releases.
```
Comment (rgacogne) [inline pdns/recursordist/docs/appendices/EOL.rst:7]: We might have too many `release` in this sentence, perhaps:
```suggestion
The latest major release train receives correctness, stability and security updates by the way of minor releases.
```

File: pdns/recursordist/docs/changelog/5.0.rst [hunk 2]
```diff
@@ -1,6 +1,84 @@
...
+    :pullreq: 13434
+
+    Make syncres code clang-tidy.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 13511
+    :tickets: 13463
+
+    Document outgoing querycounts better, including a small fix.
```
Comment (rgacogne) [inline pdns/recursordist/docs/changelog/5.0.rst:41]:
```suggestion
    Document outgoing query counts better, including a small fix.
```

File: pdns/recursordist/docs/changelog/5.0.rst [hunk 3]
```diff
@@ -1,6 +1,84 @@
...
+    :tickets: 12842
+
+    Introduce a setting to allow RPZ duplicates, including a dup handling fix.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 13497
+    :tickets: 13483
+
+    Take into account throttled queries when determining if we had a cachehit.
```
Comment (rgacogne) [inline pdns/recursordist/docs/changelog/5.0.rst:55]:
```suggestion
    Take into account throttled queries when determining if we had a cache hit.
```

File: pdns/recursordist/docs/upgrade.rst [hunk 4]
```diff
@@ -28,11 +28,17 @@ New settings
 - The :ref:`setting-tcp-threads` setting has been introduced to set the number of threads dedicated to processing incoming queries over TCP.
   Previously either the distributor thread(s) or the general worker threads would process TCP queries.
 - The :ref:`setting-qname-max-minimize-count` and :ref:`setting-qname-minimize-one-label` have been introduced to allow tuning of the parameters specified in :rfc:`9156`.
+- The :ref:`setting-allow-no-rd` has been introduced, default disabled, *disallowing* queries that do not have the ``Recursion Desired (RD)`` flag set.
+  This is a change in behviour compared to previous releases.
```
Comment (rgacogne) [inline pdns/recursordist/docs/upgrade.rst:32]:
```suggestion
  This is a change in behaviour compared to previous releases.
```

**Review by rgacogne — CHANGES_REQUESTED** [review]

File: pdns/recursordist/docs/appendices/EOL.rst [hunk 5]
```diff
@@ -3,29 +3,29 @@
 End of life statements
 ======================
 
-We aim to have a release every six months.
-The latest release receives correctness, stability and security updates.
-The two releases before that get critical updates only.
+We aim to have a major release every six months.
+The latest major release train receives correctness, stability and security updates by the way of minor releases.
+We support older releases with critical updates for one year after the next following major release.
```
Comment (neilcook) [inline pdns/recursordist/docs/appendices/EOL.rst:8]: The word 'next' seems redundant here.
```suggestion
We support older releases with critical updates for one year after the following major release.
```

**Review by aerique — APPROVED** [review]

**Pull request merged**